Lib/test/test_call.py

```diff
@@ -945,5 +945,27 @@ def test_function_with_many_args(self):
                 self.assertEqual(l['f'](*range(N)), N//2)
 
 
+class TestPEP590(unittest.TestCase):
+    def test_sys_not_callable(self):
+        import sys
+
+        self.assertFalse(callable(sys))
+        self.assertRaisesRegex(TypeError, "'module' object is not callable", sys)
```

Comment:
Some 'module' objects are now callable, maybe we can use `f"{module} object is not callable"`?

Reply:
I would suggest saying something like "Module 'os' is not callable".